kratos/processes/levelset_convection_process.h

```diff
@@ -465,7 +466,9 @@ class LevelSetConvectionProcess
 
         mpDistanceModelPart->SetProcessInfo(rBaseModelPart.pGetProcessInfo());
         mpDistanceModelPart->SetBufferSize(base_buffer_size);
-        mpDistanceModelPart->SetProperties(rBaseModelPart.pProperties());
+        for(auto i_properties = rBaseModelPart.PropertiesBegin() ; i_properties != rBaseModelPart.PropertiesEnd() ; ++i_properties){
+            mpDistanceModelPart->AddProperties(*(i_properties).base(),0);
+        }
```

Comment:
```suggestion
        for(auto it_properties = rBaseModelPart.PropertiesBegin() ; it_properties != rBaseModelPart.PropertiesEnd() ; ++it_properties){
            mpDistanceModelPart->AddProperties(*(it_properties).base());
        }
```
It is important to avoid using the mesh argument to ease the future removal of it (note that it always defaults to zero).